src/components/Climate/index.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from "react";
+import "./style.css";
+import climateUrl from './temp.svg';
+
+const Climate = ({ temperature, humidity }) => {
+    const [temp, setTemp] = useState(temperature);
+
+    const handlePlus = () => {
+        if (temp < 50) {
+            setTemp(temp + 1)
+        } else {
+            setTemp(temperature);
+        }
+    }
```

Comment:
pěkný, asi nebudeme chtít zvyšovat/zmenšovat teplotu mimo nějaké hranice :) kdybys to chtěla ještě trochu zkompaktnit, tak jde docela dobře i

```
    const handlePlus = () => {
         setTemp(temp < 50 ? temp + 1 : temp);
    }
```
a ještě víc fancy by mohlo být že když je teplota >49 tak to tlačítko nepůjde zmáčknout:
```
const canRaiseTemperature = temp < 50;

    const handlePlus = () => {
         setTemp(canRaiseTemperature ? temp + 1 : temp);
    }
...
<button className="button" disabled={canRaiseTemperature} onClick={handlePlus}>+</button>
```